charts/coredns/Chart.yaml

```diff
@@ -22,3 +22,5 @@ annotations:
   artifacthub.io/changes: |
     - kind: fixed
       description: Fixed indentation for sidecar containers
```

Comment:
Remove the previous entries, the changes section should contain only the changes related to this release